app/robots/iCubGazeboV2_5/XsensRetargeting.ini

```diff
@@ -1,17 +1,41 @@
 samplingTime            0.01
 robot                   icub
+# if yoga retargeting is enabled and the value is 'one',
+# it will do yoga retargeting, otherwise it will be walking retargeting (default is false)
+yogaRetargeting 0
+# if the following is enabled we will use smoothing with smoothingTime value (default is true)
+useSmoothing 1
 
-remote_control_boards   ("head", "torso", "left_arm", "right_arm")
+#remote_control_boards   ("head", "torso", "left_arm", "right_arm")
 # ROBOT JOINT LIST (Notice the order of the joint list is not wrong)
 # Indeed they are written according to the joint order of the walking-coordinator
-joints_list             ("neck_pitch", "neck_roll", "neck_yaw",
-                         "torso_pitch", "torso_roll", "torso_yaw",
-                         "l_shoulder_pitch", "l_shoulder_roll", "l_shoulder_yaw", "l_elbow", "l_wrist_prosup", "l_wrist_pitch", "l_wrist_yaw",
-                         "r_shoulder_pitch", "r_shoulder_roll", "r_shoulder_yaw", "r_elbow",  "r_wrist_prosup", "r_wrist_pitch", "r_wrist_yaw")
+#joints_list             ("neck_pitch", "neck_roll", "neck_yaw",
+#                         "torso_pitch", "torso_roll", "torso_yaw",
+#                         "l_shoulder_pitch", "l_shoulder_roll", "l_shoulder_yaw", "l_elbow", "l_wrist_prosup", "l_wrist_pitch", "l_wrist_yaw",
+#                         "r_shoulder_pitch", "r_shoulder_roll", "r_shoulder_yaw", "r_elbow",  "r_wrist_prosup", "r_wrist_pitch", "r_wrist_yaw")
+
 
 smoothingTime   0.25
 # The max difference (threshold) of a joint value coming from the human (rad)
 jointDifferenceThreshold 0.5
 wholeBodyJointsPort           /HumanStateWrapper/state:i
-controllerPort     /jointPosition:o
+controllerJointsPort     /jointPosition:o
+controllerCoMPort     /CoM:o
+
+# ROBOT JOINT LIST (Notice the order of the joint list is not wrong)
+# Indeed they are written according to the joint order of the walking-coordinator
+joints_list_walking_controller (
+                         "neck_pitch", "neck_roll", "neck_yaw",
+                         "torso_pitch", "torso_roll", "torso_yaw",
+                         "l_shoulder_pitch", "l_shoulder_roll", "l_shoulder_yaw", "l_elbow", "l_wrist_prosup",
+                         "r_shoulder_pitch", "r_shoulder_roll", "r_shoulder_yaw", "r_elbow",  "r_wrist_prosup" )
+
+# ROBOT JOINT LIST (Notice the order of the joint list is not wrong)
+# Indeed they are written according to the joint order of the yoga-retargeting
+joints_list_yoga_controller (
+                         "torso_pitch", "torso_roll", "torso_yaw",
+                         "l_shoulder_pitch", "l_shoulder_roll", "l_shoulder_yaw", "l_elbow",
+                         "r_shoulder_pitch", "r_shoulder_roll", "r_shoulder_yaw", "r_elbow",
+                         "l_hip_pitch", "l_hip_roll", "l_hip_yaw", "l_knee", "l_ankle_pitch", "l_ankle_roll",
+                         "r_hip_pitch", "r_hip_roll", "r_hip_yaw", "r_knee", "r_ankle_pitch", "r_ankle_roll")
```

Comment:
Having two different configuration files will help in case new  teleoperation will be implemented

Reply:
done!